src/qt/bitcoingui.cpp

```diff
@@ -714,6 +714,8 @@ void BitcoinGUI::addWallet(WalletModel* walletModel)
     WalletView* wallet_view = new WalletView(walletModel, platformStyle, walletFrame);
     if (!walletFrame->addView(wallet_view)) return;
 
+    wallet_view->setPrivacy(isPrivacyModeActivated());
```

Comment:
Why move this up top? Now it's after the signal is connected, so there's a tiny chance of a race...

Reply:
Since I moved `enableHistoryAction` within `setWalletActionsEnabled` I thought there was a evaluation of the `wallet_view` privacy there... taking that back down.

Reply:
done.